src/atwho.js

```diff
@@ -101,7 +101,9 @@ define([
                     callbacks: {
                         matcher: function(flag, subtext) {
                             var match, regexp;
-                            regexp = new RegExp('(\\s+|^)' + RegExp.escape(flag) + '([\\w_/]*)$', 'gi');
+                            // Match text that starts with the flag and then looks like a path.
+                            // CKEditor reserves the right to insert arbitrary zero-width spaces, so watch for those.
```

Comment:
That's annoying. I assume those zero-width spaces are not included in the content when getting the value from the field (so we don't need to worry about stripping them out elsewhere).